percona-toolkit/Dockerfile

```diff
@@ -1,19 +1,40 @@
-FROM debian:jessie
-MAINTAINER Percona Development <info@percona.com>
+FROM centos:7
 
-RUN apt-get update && apt-get install -y --force-yes --no-install-recommends \
-                apt-transport-https ca-certificates \
-                pwgen \
-        && rm -rf /var/lib/apt/lists/*
+LABEL name="Percona Server for MongoDB" \
+	release="3.2.0" \
+	vendor="Percona" \
+	summary="Percona Toolkit is a collection of advanced command-line tools used by Percona" \
+	description="Percona Toolkit is derived from Maatkit and Aspersa, two of the best-known toolkits for MySQL server administration. It is developed and supported by Percona." \
+	maintainer="Percona Development <info@percona.com>"
 
-RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A
+ENV PT_VERSION 3.2.0-1.el7
+ENV PS_VERSION 8.0.19-10.1.el7
 
-RUN echo 'deb https://repo.percona.com/apt jessie main' > /etc/apt/sources.list.d/percona.list
+# check repository package signature in secure way
+RUN set -ex; \
+    export GNUPGHOME="$(mktemp -d)"; \
+    gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A; \
+    gpg --batch --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona; \
+    rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7; \
+    curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm; \
+    rpmkeys --checksig /tmp/percona-release.rpm; \
+    yum install -y /tmp/percona-release.rpm; \
+    rm -rf "$GNUPGHOME" /tmp/percona-release.rpm; \
+    rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY; \
+    percona-release disable all; \
+    percona-release setup ps-80
 
-RUN  apt-get update \
-        && apt-get install -y --force-yes percona-toolkit \
-        && rm -rf /var/lib/apt/lists/* 
+RUN set -ex; \
+  yum install -y \
+    percona-toolkit-${PT_VERSION} \
```

Comment:
you need to enable ps-80 and just install percona-toolkit , percona-server-shared should be installed as dependency